cg/cli/compress/fastq.py

```diff
@@ -186,3 +188,59 @@ def decompress_spring(context, case_id, dry_run):
         return
 
     LOG.info("Decompressed spring archives in %s individuals", decompressed_inds)
+
+
+@click.command("spring-flowcell")
```

Comment:
Suggestion to add one more level instead and use commands like:

`cg decompress spring flowcell <flowcell-id>`
`cg decompress spring case <case-id>`
`cg decompress spring ticket <ticket-id>`
And perhaps even
`cg decompress spring individual <ind-id>`

Reply:
I like when commands becomes as similar to meaningful sentences as possible. Easy to understand

Reply:
Yeah this would be much better - but that would be technically a major PR then. But no problem, this can be done

Reply:
@Mropat never let semantik versioning stop you from implementing a better technical solution. After all it is just a one time change of numbers, documentation and communication. Hard to maintain and difficult to understand software stays as  a problem for a long time.

Reply:
I agree with the semantics @moonso mentioned, but as a member of prod I gladly welcome the solution created here! 
I would do:
`cg decompress spring sample` instead of `cg decompress spring individual` for analogous terms in statusdb 👍